front_end/html/summarize_logs.html

```diff
@@ -0,0 +1,158 @@
+    <div class="field is-grouped">
+      <div class="control">
+        <label class="label">Select File</label>
+        <div class="select is-info is-medium">
+          <select id="file_select" name="file_select">
+            <option selected disabled>Select File</option>
+            <option value="logs/summarized/HitsAnyUser.tsv.gz">HitsAnyUser</option>
+            <option value="logs/summarized/HitsPerUser.tsv.gz">HitsPerUser</option>
+            <option value="logs/summarized/LoadDuration.tsv.gz">LoadDuration</option>
+          </select>
+        </div>
+      </div>
+      <div class="control">
+        <label class="label">Filter Location</label>
+        <div class="select is-info is-medium">
+          <select id="filter_select" name="filter_select">
+            <option selected>No filter</option>
+            {% for item in filter_list %}
+              <option>{{ item }}</option>
+            {% end %}
+          </select>
+        </div>
+      </div>
+    </div>
+    <div class="control">
+      <label class="label">Filter by Date</label>
+      <div class="field is-grouped">
+        <div class="control">
+          <label class="label">Year</label>
+          <div class="select is-info is-medium">
+            <select id="year_select" name="year_select">
+              <option selected>No filter</option>
+              <option>20</option>
+            </select>
+          </div>
+        </div>
+        <div class="control">
+          <label class="label">Month</label>
+          <div class="select is-info is-medium">
+            <select id="month_select" name="month_select">
+              <option selected>No filter</option>
+                {% for month in months %}
+                  <option>{{ month }}</option>
+                {% end %}
+            </select>
+          </div>
+        </div>
+        <div class="control">
+          <label class="label">Day</label>
+          <div class="select is-info is-medium">
+            <select id="day_select" name="day_select">
+              <option selected>No filter</option>
+                {% for day in days %}
+                  <option>{{ day }}</option>
+                {% end %}
+            </select>
+          </div>
+        </div>
+      </div>
+    </div>
+    <input type="submit" class="button is-info is-medium is-vcentered" value="Submit"/>
+  </div>
+  <div class="column is-half">
+    <h4>Download Log Files</h4>
+    <p>
+      <p>Hits Any User <a href="/logs/summarized/HitsAnyUser.tsv.gz" download="HitsAnyUser.tsv.gz" class="button is-info is-small">Download</a></p>
+      <p>Hits Per User <a href="/logs/summarized/HitsPerUser.tsv.gz" download="HitsPerUser.tsv.gz" class="button is-info is-small">Download</a></p>
+      <p>Load Duration <a href="/logs/summarized/LoadDuration.tsv.gz" download="LoadDuration.tsv.gz" class="button is-info is-small">Download</a></p>
+    </p>
+  </div>
+</div>
+
+
+<script type="text/javascript">
```

Comment:
I'm not sure if I understand what this JavaScript is doing. Is it still needed?

Reply:
Yes, it keeps an option selected after the form is submitted. Without this, the dropdown would reset every time the submit button is clicked.